automerge-backend-wasm/Cargo.toml

```diff
@@ -35,12 +35,16 @@ serde_bytes = "0.11.5"
 version = "^0.2"
 features = ["serde-serialize"]
 
+[package.metadata.wasm-pack.profile.release]
+wasm-opt = false
+
```

Comment:
Is this related?

Reply:
Good catch...  I needed that to get the tests to pass on my new M1 MacBook but it should not be in the PR!